tests/dockerfiles/buildpack_deps_jessie_arg

```diff
@@ -0,0 +1,19 @@
+ARG version=jessie
+FROM debian:$version
```

Comment:
@ForgetMe17 does this work when `{}` are used too? i.e. `FROM debian:${version}`

Reply:
Yes, i use `replace_env` to replace `$xxx` and `${xxx}`. This function has already been tested in `tests\test_analyze_docker_dockerfile.py`